modules/kmm-example-cr.adoc

```diff
@@ -8,22 +8,15 @@
 
 This section shows an example of the `PreflightValidationOCP` resource in the YAML format.
 
-The example verifies all the currently present modules against the upcoming kernel version included in the {product-title} release 4.11.18, which the following release image points to:
-
-[source,terminal]
-----
-quay.io/openshift-release-dev/ocp-release@sha256:22e149142517dfccb47be828f012659b1ccf71d26620e6f62468c264a7ce7863
-----
-
-Because `.spec.pushBuiltImage` is set to `true`, KMM pushes the resulting images of Build/Sign into the defined repositories.
+The example verifies all the currently present modules against the upcoming `5.8.18-101.fc31.x86_64` kernel, and push the resulting images of Build/Sign in to the defined repositories.
 
 [source,yaml]
 ----
-apiVersion: kmm.sigs.x-k8s.io/v1beta1
+apiVersion: kmm.sigs.x-k8s.io/v1beta2
 kind: PreflightValidationOCP
 metadata:
- name: preflight
+  name: preflight
 spec:
- releaseImage: quay.io/openshift-release-dev/ocp-release@sha256:22e149142517dfccb47be828f012659b1ccf71d26620e6f62468c264a7ce7863
- pushBuiltImage: true
+  kernelVersion: 5.8.18-101.fc31.x86_64
```

Comment:
`PreflightValidationOCP` uses `releaseImage` in its spec, not `kernelVersion`. I would also stick to the existing content here - although increasing the indentation by 1 space is correct.

Reply:
Please verify this change.